36-QWERTY-Tap-Switch-Keymap

ShamalLakshan opened the pull request:
Added new keymap and necessary details

precondition commented on src/posts/keymaps/ShamalLakshan.md:
```diff
@@ -0,0 +1,24 @@
+keybindings: []                                                                                                                 # [Vim, Emacs, Kakoune, Graphics/CAD, TWM, Spreadsheets, Gaming]
+keyboard: BGKeeb                                                                                                                # Kyria or Corne or Dactyl Manuform 5x6 or ...
+keyCount: 36
+keymapImage: "https://github.com/ShamalLakshan/36-QWERTY-Tap-Switch-Keymap/blob/main/wallpapers/wallpaper05.png"
```
I believe it would be better to vertically stack the images of the layers you have in the `images/` folder. With such a wide image, all you see from the KeymapDB card is the girl.

<img width="375" height="482" alt="KeymapDB card of ShamalLakshan's wallpaper keymap" src="https://github.com/user-attachments/assets/66e54309-8b0f-48ce-a89a-f2e658d697a6" />

ShamalLakshan replied:
Yes please, if you can, it will be really helpful since I don't know muck about the internal build system used on the site....

ShamalLakshan commented:
Created a new preview image and updated the PR.
New Image: https://raw.githubusercontent.com/ShamalLakshan/36-QWERTY-Tap-Switch-Keymap/refs/heads/main/images/preview.png
Added in: 930cf9f880a6ccb7f3429452eaeaa316f207571c

precondition commented:
That's even better! 

Thanks.

Latest commit at date of merge: https://github.com/ShamalLakshan/36-QWERTY-Tap-Switch-Keymap/commit/1efc688800011fc88774b172f8b142bd6e693791
